src/controllers/resource-controller.js

```diff
@@ -0,0 +1,105 @@
+const {responseServer, responseGeneral, responseNotFound} = require('../helpers/response-result');
+const {StatusCodes} = require ('http-status-codes');
+const Accounts = require('../models/accounts');
+const Lessons = require('../models/lessons');
+const Questions = require('../models/questions');
+const path = require('path');
+const fs = require('fs').promises;
+
+const patchResource = (request, response, next) => {
+    const URL = request.body.URL;
+    fs.stat(__dirname+URL)
+    .then( ()=>{
+        response.sendFile(__dirname+URL);
+    })
+    .catch((error) =>{
+        if(error.code == 'ENOENT'){
+            responseNotFound(response);
+        }
+        else{
+            responseServer(response, error);
+        } 
+    });
+}
+
+const postImageLesson = (request, response, next) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
const postImageLesson = (request, response, next) => {
```